docs/source/open-software-summer-school/2026/extracellular-electrophysiology.md

```diff
@@ -0,0 +1,108 @@
+(track-extracellular-ephys)=
+# Track: Extracellular Electrophysiology
+
+Alongside the development of high-density recording probes (e.g. [Neuropixels](https://www.neuropixels.org/), 
+the range and  complexity of extracellular preprocessing methods has greatly increased in recent years. 
+In this course, we will cover the theory and practical implementation of the full extracellular
+processing pipeline, including preprocessing, sorting and quality control. 
+
+We will use [SpikeInterface](https://spikeinterface.readthedocs.io/en/stable/) 
+to implement the pipeline, with manual curation in the [SpikeInterface GUI](https://github.com/SpikeInterface/spikeinterface-gui). 
+We will explore modern toolkits for sorting quality assessment ([Bombcell](https://github.com/Julie-Fabre/bombcell)), 
+unit matching ([UnitMatch](https://github.com/EnnyvanBeest/UnitMatch)) and downstream analysis ([PyNapple](https://github.com/pynapple-org/pynapple)).
+
+::: {admonition} Target audience
+:class: note
+
+This course is suitable for researchers and students with no experience in extracellular electrophysiology.
+However, experienced users interested in using SpikeInterface or exploring the theory of processing steps 
+in more detail will also benefit.
+
+This course is focused on analysing high-density recordings (e.g. Neuropixels).
+:::
+
+## Course overview
+
+**Monday — Introduction**
+
+_Morning_  
```

Comment:
These secondary headings don't stand out much. Could the daily headings be made into H3, and then these just bold?